apps/cookielifetime.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# shellcheck disable=2034,2059
+true
+# shellcheck source=lib.sh
+. <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh)
+
+# Check for errors + debug code and abort if something isn't right
+# 1 = ON
+# 0 = OFF
+DEBUG=0
+debug_mode
+
+# Check if root
+root_check
+
+COOKIE_LIFETIME=$(whiptail --radiolist  "Configure after what time (in seconds) after every Login every Nextcloud user gets logged out in the Browser\nSelect one with the [ARROW] Keys and the [SPACE] key and confirm by pressing [ENTER]" "$WT_HEIGHT" "$WT_WIDTH" 4 \
+"1800s" "half an hour" ON \
+"7200s" "two hours" OFF \
+"43200s" "half a day" OFF \
+"172800s" "two days" OFF \
+"604800s" "one week" OFF \
+"2419200s" "four weeks" OFF \
```

Comment:
what do you think about showing the seconds here? shall I remove them?

Reply:
Looks good to me, have to test this first. Maybe I change my mind later :)